app/views/govuk_publishing_components/components/_govspeak.html.erb

```diff
@@ -1,5 +1,6 @@
 <%
   add_gem_component_stylesheet("govspeak")
+  add_gem_component_stylesheet("details")
```

Comment:
@MartinJJones this solves the problem for now but isn't required under normal circumstances for the attachment component to correctly render the details component. Is this something better addressed in govspeak?

Reply:
Yeah, I think it would be best to fix this in govspeak.

Instead of including the additional stylesheet here, we could always import the `details` stylesheet into `/govspeak/attachment.scss` - https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/govspeak/_attachment.scss#L13-L14 which appears to follow the current convention, for example:

```
  @import "../attachment-link";
  @import "../attachment";
  @import "../details";
```

This would mean that we do not request an additional stylesheet when the details component is not used, however I would expect this to slightly increase the overall size of the govspeak.css

I still think this approach makes sense however, for example if an application does not use the individual loading of stylesheets feature but needs to render the `attachment` component, we would likely face the same issue with styling, in that we would still need to remember to include the `details` component styles as well, we effectively would do the same here but for Govspeak.

Reply:
Thanks @MartinJJones. I've made and tested your suggestion, works fine, makes sense 👍